book/src/pong-tutorial.md

```diff
@@ -26,9 +28,15 @@ let path = format!(
 
 But for your own project you'll probably want something like this:
 
-```rust,ignore
+```rust,norun
 let path = "./resources/display_config.ron";
 ```
 
+The reason `application_root_dir` is used in our examples is that the
+required resources are not in the working directory, but rather somewhere
+in the `examples` folder. However, in an actual game it would be bad to
+hardcode the path of the resources to the directory that was used to compile
+the application.
```

Comment:
Could you clarify _why_ it would be bad? What would happen?

Reply:
Yeah, I somehow assumed the last step is just logical. Really gotta learn explaining things better.

Reply:
I'm super happy that you are working over deficiencies in the docs and book as you see them!